### PR TITLE
[DBIO-1401] Fix: service image mismatch lab

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-1401

### Changelog/Description 🗒️
- fix service image mismatch on lab

### WARNING ⚠️
-

### CODE REVIEW

File: src/views/Dashboard/Lab/Services/index.vue [hunk 1]
```diff
@@ -189,7 +192,7 @@ export default {
       if(val && val != ""){
          return val
       }
-      return "https://ipfs.io/ipfs/QmaGr6N6vdcS13xBUT4hK8mr7uxCJc7k65Hp9tyTkvxfEr"
+      return "https://ipfs.io/ipfs/QmZveuvcQAbkcbcop28iqWrnCMoxN2n5SGzA3Hwr1cJ61i/debio-logo.png"
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/Services/index.vue:195]: The DeBio logo shouldn't be hardcoded like this. Wouldn't it be better to turn it into an Icon? @luthfiazhari.

Please ask @bukandicki for help. @bukandicki, please help @luthfiazhari with this.

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/Services/index.vue (hunk 1 above)
Comment (agustinustheo) [reply src/views/Dashboard/Lab/Services/index.vue:195]: Example: https://github.com/debionetwork/debio-customer-ui/pull/309/commits/1c248c86dc174be12e1d4b3a1e4a55791a4ca96a

**Comment by agustinustheo** [comment]
Closing because of inaction @luthfiazhari

**Comment by luthfiazhari** [comment]
this PR is not relevan again with current task.

**Pull request closed without merging**